packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -37,6 +40,14 @@ interface IButtonClassNames {
 
 export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState> implements IButton {
 
+  private get _isSplitButton(): boolean {
```

Comment:
The props should probably contain a comment describing that this particular combination turns the element into a split button.